packages/components/src/components/TableComponent/TableComponent.scss

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '~carbon-components/scss/globals/scss/vars';
+
+.tableComponent {
+  .bx--table-toolbar {
+    background: transparent;
+  }
+
+  .bx--data-table th {
+    background-color: $ui-background;
+  }
+
+  .bx--data-table th.bx--table-column-checkbox {
+    background-color: $ui-background;
+  }
+
+  .bx--data-table tbody td.cellText {
+    padding-left: 22.5px;
```

Comment:
what's this for?

Reply:
Don't know if you remember, but for some reason in the past the table row cells text weren't being aligned with its respective column cell so this padding fixes that.